File: src/main/java/com/github/Doomsdayrs/Jikan4java/types/Main/Anime/Episodes/Episode.java
package com.github.Doomsdayrs.Jikan4java.types.Main.Anime.Episodes;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.github.Doomsdayrs.Jikan4java.types.Main.Anime.Aired;

/*
 * Licensed to the Apache Software Foundation (ASF) under one
 * or more contributor license agreements.  See the NOTICE file
 * distributed with this work for additional information
 * regarding copyright ownership.  The ASF licenses this file
 * to you under the Apache License, Version 2.0 (the
 * "License"); you may not use this file except in compliance
 * with the License.  You may obtain a copy of the License at
 *
 *   http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing,
 * software distributed under the License is distributed on an
 * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 * KIND, either express or implied.  See the License for the
 * specific language governing permissions and limitations
 * under the License.
 * ====================================================================
 * Jikan4java
 * 29 / October / 2018
 *
 * @author github.com/doomsdayrs
 */
@JsonFormat(shape = JsonFormat.Shape.OBJECT)
public class Episode {
    @JsonProperty("episode_id")
    public int episode_id;

    @JsonProperty("title")
    public String title;

    @JsonProperty("title_japanese")
    public String title_japanese;

    @JsonProperty("title_romanji")
    public String title_romanji;

    @JsonProperty("aired")
    public Aired aired;

    @JsonProperty("filler")
    public boolean filler;

    @JsonProperty("recap")
    public boolean recap;

    @JsonProperty("video_url")
    public String video_url;

    @JsonProperty("forum_url")
    public String forum_url;
    

    @Override
    public String toString() {
        return "Episode{" +
                "episode_id=" + episode_id +
                ", title='" + title + '\'' +
                ", title_japanese='" + title_japanese + '\'' +
                ", title_romanji='" + title_romanji + '\'' +
                ", aired=" + aired +
                ", filler=" + filler +
                ", recap=" + recap +
                ", video_url='" + video_url + '\'' +
                ", forum_url='" + forum_url + '\'' +
                '}';
    }
}

File: src/main/java/com/github/doomsdayrs/jikan4java/model/main/anime/episodes/Episode.kt
package com.github.doomsdayrs.jikan4java.model.main.anime.episodes;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonProperty;

/*
 * This file is part of Jikan4java.
 *
 * Jikan4java is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * Jikan4java is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with Jikan4java.  If not, see <https://www.gnu.org/licenses/>.
 * ====================================================================
 * Jikan4java
 * 29 / October / 2018
 *
 * @author github.com/doomsdayrs
 */
@JsonFormat(shape = JsonFormat.Shape.OBJECT)
public class Episode {
    @JsonProperty("episode_id")
    @SuppressWarnings("WeakerAccess")
    public int episode_id;

    @JsonProperty("title")
    public String title;

    @JsonProperty("title_japanese")
    public String title_japanese;

    @JsonProperty("title_romanji")
    @SuppressWarnings("WeakerAccess")
    public String title_romanji;

    @JsonProperty("aired")
    public String aired;
    //public Aired aired;

    @JsonProperty("filler")
    @SuppressWarnings("WeakerAccess")
    public boolean filler;

    @JsonProperty("recap")
    @SuppressWarnings("WeakerAccess")
    public boolean recap;

    @JsonProperty("video_url")
    public String video_url;

    @JsonProperty("forum_url")
    @SuppressWarnings("WeakerAccess")
    public String forum_url;
    

    @Override
    public String toString() {
        return "Episode{" +
                "episode_id=" + episode_id +
                ", title='" + title + '\'' +
                ", title_japanese='" + title_japanese + '\'' +
                ", title_romanji='" + title_romanji + '\'' +
                ", aired=" + aired +
                ", filler=" + filler +
                ", recap=" + recap +
                ", video_url='" + video_url + '\'' +
                ", forum_url='" + forum_url + '\'' +
                '}';
    }
}
